Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -4626,63 +4631,140 @@ private bool DoGameEnd(ITeam team)
                         var deathmessage = "";
                         switch (alives.Count())
                         {
+                            case 4:
+                                // Tanner and sorcerer and thief and doppelganger, tanner will die.
+                                if (alives.Any(x => x.PlayerRole == IRole.Tanner) && alives.Any(x => x.PlayerRole == IRole.Sorcerer) && alives.Any(x => x.PlayerRole == IRole.Thief) && alives.Any(x => x.PlayerRole == IRole.Doppelgänger))
```

Comment:
This doesn't really look nice. The line length becomes way too long. It could be shortened as:
`if (new[] { IRole.Tanner, IRole.Sorcerer, IRole.Thief, IRole.Doppelgänger }.All(x => alives.Any(y => y.PlayerRole == x)))`

Anyway, it won't be needed (see the previous comment).